docs/tsconfig.json

```diff
@@ -4,13 +4,14 @@
   "compilerOptions": {
     "baseUrl": ".",
     "paths": {
-      "@/*": ["src/*"]
+      "@/*": ["src/*"],
+      "@shadcn/*": ["src/components/ui/*"]
```

Comment:
**thought:** not sure how i feel about this...

i think it's nice, but it does obscure the import

if you look at [shadcn itself](https://ui.shadcn.com/docs/components/accordion#usage), it always imports from `@/components/ui`

Reply:
I agree; it was an idea I had to make it more straightforward for junior contributors to understand where these components come from if they want to change or add new ones. I feel that Shadcn does a lousy job documenting itself in a project. I can see how a project deteriorates in quality as people who don't know what Shadcn is start mucking around with the copied components.